src/diffusers/pipelines/__init__.py

```diff
@@ -170,6 +170,7 @@
     _import_structure["controlnet_sd3"].extend(
```

Comment:
The `__init__.py` files looks correct. I think you will have to update one more file: `src/diffusers/__init__.py`